src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -50,7 +53,7 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
     // check if tracing is completely disabled
     if (IsTracingDisabled())
     {
-        Info("DATADOG TRACER DIAGNOSTICS - Profiler disabled in ", environment::tracing_enabled);
+        Logger::Logger::Info("DATADOG TRACER DIAGNOSTICS - Profiler disabled in ", environment::tracing_enabled);
```

Comment:
`Logger::Logger::Info` here and `Logger::Info` in the same file, what's the difference?

Reply:
Oops

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1625/commits/d419680b0e56138ce4a9e164f75155a83b7516c2